front/app/containers/Admin/initiatives/settings/index.tsx

```diff
@@ -5,10 +5,12 @@ import { isNilOrError } from 'utils/helperUtils';
 // hooks
```

Comment:
This component was updated to include the `PageBody` component above. When the form is submitted, it also has to update the page with the `updatePage` service.